sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDB.scala

```diff
@@ -316,6 +321,25 @@ class RocksDB(
     }
   }
 
+  def merge(key: Array[Byte], value: Array[Byte],
```

Comment:
Could we add a function level comment here ?

Reply:
Yes, thanks for catching. Added